lib/models/embed_field.cpp

```diff
@@ -0,0 +1,61 @@
+#include <models/embed_field.hpp>
+

```

Comment:
_inline here is an optional but is being treated like it isnt. this is because if the user doesnt set it, e.g.:
```
disccord::models::embed an_embed;
an_embed.decode(web::json::value::parse("{\"name\":\"roflcopter\",\value"\":\"supyo\"}"));
```

we will still set it "inline" to false and things won't break, but they can also set it themselves which will overwrite our default in the `decode` method.